pkg/apis/kops/networking.go

```diff
@@ -195,80 +195,224 @@ const CiliumIpamEni = "eni"
+	// Setting this has currently no effect.
+	SingleClusterRoute bool `json:"singleClusterRoute,omitempty"`
+	// SocketPath has not been implemented in the new templates and may be removed in the future.
+	// Setting this has currently no effect.
+	SocketPath string `json:"socketPath,omitempty"`
+	// StateDir has not been implemented in the new templates and may be removed in the future.
+	// Setting this has currently no effect.
+	StateDir string `json:"stateDir,omitempty"`
+	// TracePayloadLen has not been implemented in the new templates and may be removed in the future.
+	// Setting this has currently no effect.
+	TracePayloadLen int `json:"tracePayloadlen,omitempty"`
+	// Tunnel determines the cilium tunelling mode. Possible values are "vxlan", "geneve", or "disabled".
+	// Default: vxlan
+	Tunnel string `json:"tunnel,omitempty"`
+	// EnableIpv6 enables cluster ipv6 traffic. Note that if both EnableIpv6 and EnableIpv4 are set to false.
+	// Ipv4 will be enabled.
+	// Default: false
+	EnableIpv6 bool `json:"enableipv6"`
+	// EnableIpv4 enables cluster ipv4 traffic. Note that if both EnableIpv6 and EnableIpv4 are set to false.
+	// Ipv4 will be enabled.
+	// Default: false
+	EnableIpv4 bool `json:"enableipv4"`
+	// MonitorAggregation sets the level of packet monitoring. Can be set to either "low", "medium", or "maximum".
+	// Default: medium
+	MonitorAggregation string `json:"monitorAggregation"`
+	// BPFCTGlobalTCPMax is the maximum number of entries in TCP CT table.
+	// Default: 524288
+	BPFCTGlobalTCPMax int `json:"bpfCTGlobalTCPMax"`
+	// BPFCTGlobalAnyMax is the maximum number of entries in non-TCP CT table.
+	// Default: 262144
+	BPFCTGlobalAnyMax int `json:"bpfCTGlobalAnyMax"`
+	// PreallocateBPFMaps reduce the per-packet latency at the expense of up-front memory allocation.
+	// Default: true
+	PreallocateBPFMaps bool `json:"preallocateBPFMaps"`
+	// SidecarIstioProxyImage is the regular expression matching compatible Istio sidecar istio-proxy.
+	// container image names.
+	// Default: cilium/istio_proxy
 	SidecarIstioProxyImage string `json:"sidecarIstioProxyImage"`
-	ClusterName            string `json:"clusterName"`
-	ToFqdnsEnablePoller    bool   `json:"toFqdnsEnablePoller"`
+	// ClusterName is the of the cluster and is only relevant when building a mesh of clusters.
+	ClusterName string `json:"clusterName"`
+	// ToFqdnsEnablePoller is disabled by default starting from version 1.4.x in favor
+	// of a more powerful DNS proxy-based implementation.
+	// Enable this option if you want to use FQDN policies but do not want to use
+	// the DNS proxy.
+	// To ease upgrade, users may opt to set this option to "true".
+	// Default: true
+	ToFqdnsEnablePoller bool `json:"toFqdnsEnablePoller"`
+	// ContainerRuntimeLabels determines the container runtime(s) used by Cilium
+	// Supported values are: "none", "containerd", "crio", "docker", "auto"
+	// Default: none
```

Comment:
If multiple values are possible, how are they given? Comma separated?

Reply:
I am wondering if I should remove this part. Cilium 1.7.0 ignores this setting anyway, and the argument will be removed in 1.8.0